boards/arm/lpcxpresso55s69/doc/index.rst

```diff
@@ -191,6 +195,20 @@ see the following message in the terminal:
    ***** Booting Zephyr OS v1.14.0 *****
    Hello World! lpcxpresso55s69_cpu0
 
+Building and flashing secure/non-secure with Arm |reg| TrustZone |reg|
+---------------------------------------------------------------------
+The TF-M integration samples can be run by a LPC55S69, using the ``lpcxpresso55s69_ns`` target.
+For we build TFM wihout a bootloader (MCUboot), to run TFM images we need to flash them manually.
```

Comment:
This sentence is unclear. See the other comments to possibly clarify.

Reply:
@microbuilder please check this documentation again?